schemes/default.nix

```
@@ -143,6 +143,7 @@
   icy = import ./icy/icy.nix;
 
   katy = import ./katy/katy.nix;
+  kanagawa = import ./kanagawa/kanagawa.nix;
```

```suggestion

  kanagawa = import ./kanagawa/kanagawa.nix;
```
This should not be grouped together with katy